page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,155 @@
+### jQuery Fundamentals
+
+This tutorial will help you try out some common uses of jQuery.
+
+If you haven't setup a demo HTML page that includes a copy of jQuery, see [How To Setup jQuery](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/) for the complete tutorial on how to get started, or simply follow along with the examples below.
+
+### Adding and Removing an HTML Class
+
+jQuery is often used to bind events to elements in the DOM, assigning them functionality to be triggered by a user's actions on the page. To demonstrate, let's highlight the menu item a user last clicked by changing its HTML class to "active", and assigning the "active" class a different appearance using CSS.
+
+First, copy or create the `index.html` file below:
```

Comment:
Something like jsbin would be excellent because the tutorial gets bogged down in small instructions that aren’t getting to the real point of the explanation. I struggled with keeping this kind of instruction brief while still keeping it adequate, and still wasn’t satisfied. Since the first tutorial is about setting up jQuery locally, I think it makes sense to keep the instructions for setting up the files yourself and running it in the browser. But the second tutorial could skip this entirely and go straight into the content, which would free it up a lot. The jQuery API docs have an embedded demo. I like that it’s embedded, but it’s sad that you can’t change the code and mess around with it, which is a lot of what I get out of tools like jsbin.